playground/.env.example

```diff
@@ -0,0 +1 @@
+CLOUDINARY_CLOUD_NAME=ddtc5sowb
```

Comment:
if you put in your cloud name - people will copy and paste it without thinking haha id put <Your Cloud Name>

Reply:
I will change it to something different :)